test/test_prototype_transforms_functional.py

```diff
@@ -426,27 +426,74 @@ def erase_image_tensor():
         yield ArgsKwargs(image, i=1, j=2, h=6, w=7, v=torch.rand(c, 6, 7))
 
 
+_KERNEL_TYPES = {"_image_tensor", "_image_pil", "_mask", "_bounding_box", "_label"}
+
+
+def _distinct_callables(callable_names):
```

Comment:
Quite a messy method to achieve want we want. The main problem is that not all aliases need to be excluded. Some are currently used to import things from `_FT` or `_FP` so we can't rely on `method.__name__`. So we need to have an over-engineered logic to filter things out. Perhaps that's not worth it and we should just  accept the dups. It's likely we will need a better way to discriminate between Mid-level and low-level kernels. Maybe a registration API or maybe a last minute restructuring of the definitions will do it.